release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -0,0 +1,1538 @@
+//Telco Edge / TALO
+//Telco Edge / ZTP
+[discrete]
+[id="ocp-4-15-api-auth-bug-fixes"]
+==== API Server and Authentication
+
+[discrete]
+[id="ocp-4-15-bare-metal-hardware-bug-fixes"]
+==== Bare Metal Hardware Provisioning
+
+[discrete]
+[id="ocp-4-15-builds-bug-fixes"]
+==== Builds
+
+[discrete]
+[id="ocp-4-15-cloud-compute-bug-fixes"]
+==== Cloud Compute
+
+[discrete]
+[id="ocp-4-15-cloud-cred-operator-bug-fixes"]
+==== Cloud Credential Operator
+
+[discrete]
+[id="ocp-4-15-cluster-version-operator-bug-fixes"]
+==== Cluster Version Operator
+
+[discrete]
+[id="ocp-4-15-dev-console-bug-fixes"]
+==== Developer Console
+
+[discrete]
+[id="ocp-4-15-cloud-etcd-operator-bug-fixes"]
+==== etcd Cluster Operator
+
+
+[discrete]
+[id="ocp-4-15-hosted-control-plane-bug-fixes"]
+==== Hosted Control Plane
+
+[discrete]
+[id="ocp-4-15-image-registry-bug-fixes"]
+==== Image Registry
+
+
+[discrete]
+[id="ocp-4-15-installer-bug-fixes"]
+==== Installer
+
+[discrete]
+[id="ocp-4-15-kube-controller-bug-fixes"]
+==== Kubernetes Controller Manager
+
+[discrete]
+[id="ocp-4-15-kube-scheduler-bug-fixes"]
+==== Kubernetes Scheduler
+
+
+[discrete]
+[id="ocp-4-15-machine-config-operator-bug-fixes"]
+==== Machine Config Operator
+
+[discrete]
+[id="ocp-4-15-management-console-bug-fixes"]
+==== Management Console
+
+[discrete]
+[id="ocp-4-15-monitoring-bug-fixes"]
+==== Monitoring
+
+[discrete]
+[id="ocp-4-15-networking-bug-fixes"]
+==== Networking
+
+[discrete]
+[id="ocp-4-15-node-bug-fixes"]
+==== Node
+
+[discrete]
+[id="ocp-4-15-node-tuning-operator-bug-fixes"]
+==== Node Tuning Operator (NTO)
+
+[discrete]
+[id="ocp-4-15-openshift-cli-bug-fixes"]
+==== OpenShift CLI (oc)
+
+[discrete]
+[id="ocp-4-15-olm-bug-fixes"]
+==== Operator Lifecycle Manager (OLM)
+
+[discrete]
+[id="ocp-4-15-openshift-api-server-bug-fixes"]
+==== OpenShift API server
+
+[discrete]
+[id="ocp-4-15-rhcos-bug-fixes"]
+==== {op-system-first}
+
+[discrete]
+[id="ocp-4-15-scalability-and-performance-bug-fixes"]
+==== Scalability and performance
+
+[discrete]
+[id="ocp-4-15-storage-bug-fixes"]
+==== Storage
+
+[discrete]
+[id="ocp-4-15-windows-containers-bug-fixes"]
+==== Windows containers
+// Added after OCP GA
+
+[id="ocp-4-15-technology-preview"]
+== Technology Preview features
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the following tables, features are marked with the following statuses:
+
+* _Technology Preview_
+* _General Availability_
+* _Not Available_
+* _Deprecated_
+
+[discrete]
+=== Networking Technology Preview features
+
+.Networking Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.12 |4.13 |4.14
+
+|PTP dual NIC hardware configured as boundary clock
```

Comment:
Anything that is 3 GAs gets removed from the tables. Same for Rem in the dep/rem tables. 3 `Remove` and it is omitted.